app/views/strategies/index.html.haml

```diff
@@ -0,0 +1,27 @@
+%h1 Listing strategies
+
+%table.table.table-bordered
+  %thead
+    %tr
+      %th{ scope: 'col' } Name
+      %th{ scope: 'col' } Time range
+      %th{ scope: 'col' } Instrument
+      %th{ scope: 'col' } Strategy Logic
+      %th{ scope: 'col' }
+      %th{ scope: 'col' }
+      %th{ scope: 'col' }
+
+  %tbody
+    - @strategies.each do |strategy|
```

Comment:
Look at Motohelper. You can replace this with partial '_strategy' (= render @strategies). 
https://github.com/workshop-team/motohelper/blob/master/app/views/maintenances/index.html.haml